src/components/molecules/UserSearchBar/UserSearchBar.tsx

```diff
@@ -18,20 +18,20 @@ const UserSearchBar: FC<UserSearchBarProps> = ({ onSelect }) => {
     WithId<User>
   >();
 
-  const partygoers = usePartygoers();
+  const { recentWorldUsers } = useRecentWorldUsers();
```

Comment:
Are we 100% sure that only the recent users should be searched for?

Reply:
Really good catch. Do you think there are other such places?

Reply:
`NavSearchBar.tsx` is also searching for users maybe we should use this in there as well.

Reply:
Looking at https://github.com/sparkletown/sparkle/pull/1047/commits/591678d40817b5978f675635739a11a624594646 and https://github.com/sparkletown/sparkle/pull/1047/files?file-filters%5B%5D=.tsx#diff-e1a64d67f174af23ebf6e3124ae2491e09824b4e81eaf2bf24348343024f39c8 this seems to be done now